### DISCUSSION TITLE
Ability to set Maps or Stable-master map as main mode

### DISCUSSION THREAD

**Discussion by dt192** [wduda/TravelWindowII#239] [Feature Ideas & Requests]
It would be nice to be able to set maps* as my main view, as I can currently with list view, icon view etc., so I click the icon and it opens straight to the map view.

*once the drop down is added to enable swapping between them.

**Comment by wduda** [comment]
Will be implemented in [v4.0.0 #265](https://github.com/wduda/TravelWindowII/pull/265)
